src/SFA.DAS.LevyTransferMatching.Web/Controllers/OpportunitiesController.cs

```diff
@@ -27,12 +27,19 @@ public async Task<IActionResult> Index()
             var viewModel = await _opportunitiesOrchestrator.GetIndexViewModel();
             return View(viewModel);
         }
-        
+
+        [HttpPost]
+        public async Task<IActionResult> Index(IndexRequest request)
```

Comment:
We needn't do a `POST` here - we can change the form action to `GET` and just add the `IndexRequest` to the existing action. That way we don't have to render a view off a `POST`, which causes cache-expiry type issues.